### ISSUE TITLE
Error creating section named [default]

### ISSUE THREAD

**Issue by fawzibr** [pixelb/crudini#107]
Tried to create a section named default using **crudini --set myfile.ini "default"**. I get the following error:

```
Traceback (most recent call last):
  File "/root/.local/bin/crudini", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/root/.local/share/pipx/venvs/crudini/lib/python3.12/site-packages/crudini.py", line 1288, in main
    return crudini.run()
           ^^^^^^^^^^^^^
  File "/root/.local/share/pipx/venvs/crudini/lib/python3.12/site-packages/crudini.py", line 1174, in run
    self.command_set()
  File "/root/.local/share/pipx/venvs/crudini/lib/python3.12/site-packages/crudini.py", line 1013, in command_set
    self.set_name_value(self.section, self.param, self.value)
  File "/root/.local/share/pipx/venvs/crudini/lib/python3.12/site-packages/crudini.py", line 969, in set_name_value
    self.conf.add_section(section)
  File "/root/.local/share/pipx/venvs/crudini/lib/python3.12/site-packages/iniparse/compat.py", line 62, in add_section
    raise ValueError('Invalid section name: %s' % section)
ValueError: Invalid section name: default

```

I can create "DEFAULT" fine, any other variation (default, Default,deFault,...) fails. any reason for that?

**Comment by fawzibr** [comment]
Ah, didn't saw that it was an iniparse problem. Just tried to create a section named "default" and didn't understand why it was failing.

**Comment by pixelb** [comment]
OK I'll fix issue #80 to just create a "default" section with normal properties, and not map "default" to "DEFAULT"
